addon-test-support/run-options.ts

```diff
@@ -0,0 +1,32 @@
+import { RunOptions } from 'axe-core';
+import QUnit from 'qunit';
+import { getContext } from '@ember/test-helpers';
+
+let _runOptions = new WeakMap<object, RunOptions>();
+
+/**
+ * Sets run options specific to a test.
+ *
+ * @param options Axe {RunOptions} to be provided to the audit helper.
+ */
+export function setRunOptions(options: RunOptions = {}) {
+  _runOptions.set(getContext(), options);
+}
+
+/**
+ * Gets run options specific to a test.
+ *
+ * @param context Test context object, accessed using `@ember/test-helpers` `getContext` function.
+ */
+export function getRunOptions(context: object) {
+  return _runOptions.get(context);
+}
```

Comment:
I think I'd prefer to use a stack like system here. What do you think?

Reply:
I like this. This makes perfect sense to me in that the top of the stack will contain the specific options we're using. So, the algo will look like the following:

calling `setRunOptions`...

- in `tests/test-helper.js` pushes default options onto the stack. If nothing else pushes options, the defaults are used
- in test module's `beforeEach`, pushes specific options onto the stack. If nothing else pushes options, these options are used. These options need to be popped off the stack once the module is done
- test pushes specific options onto the stack. Uses these options, as they're the top of the stack. These options need to be popped off the stack once the test is done.